package.json

```diff
@@ -36,13 +36,25 @@
     "@ngrx/effects": "^2.0.3",
     "@ngrx/store": "^2.2.2",
     "angular2-jwt": "0.1.25",
-    "cordova-custom-config": "4.0.2",
+    "cordova-custom-config": "~3.1.4",
+    "cordova-ios": "4.5.1",
     "ionic-angular": "3.7.1",
     "ionicons": "3.0.0",
     "raven-js": "^3.18.1",
     "rxjs": "5.4.0",
     "sw-toolbox": "3.6.0",
-    "zone.js": "0.8.18"
+    "zone.js": "0.8.18",
+    "cordova-android": "~6.1.2",
+    "ionic-plugin-keyboard": "~2.2.1",
+    "cordova-plugin-whitelist": "1.3.1",
+    "cordova-plugin-console": "1.0.5",
+    "cordova-plugin-statusbar": "2.2.1",
+    "cordova-plugin-device": "1.1.4",
+    "cordova-plugin-splashscreen": "~4.0.1",
+    "cordova-plugin-x-toast": "~2.6.0",
+    "phonegap-plugin-barcodescanner": "~6.0.8",
+    "cordova-plugin-inappbrowser": "~1.7.1",
+    "cordova-plugin-wkwebview-engine": "^2.2.0"
```

Comment:
Travis fails because of
> No compatible version found: cordova-plugin-wkwebview-engine@^2.2.0

Can you check to make sure this is the right version?